x-element.js

```diff
@@ -1074,12 +1074,14 @@ class TemplateEngine {
       const result = TemplateEngine.#resultMap.get(resultReference);
       if (TemplateEngine.#cannotReuseResult(state.result, result)) {
         TemplateEngine.#removeWithin(container);
-        state.result = result;
+        TemplateEngine.#ready(result);
+        TemplateEngine.#commit(result);
         TemplateEngine.#inject(result, container);
```

Comment:
The above three lines sum up the change neatly. Before, we readied / injected in one line… then committed the update. Now, we ready, commit, and _lastly_ inject.

Yes, in the future we will change all these internal method names… not now though.